CONTRIBUTING.MD

```diff
@@ -0,0 +1,27 @@
+# Contributing a Devfile Stack to this registry
+
+This document outlines the requirements for contributing a devfile stack to this repository
+
+## Steps
+
+1) Verify your Devfile stack functions with odo.
+  
+    - Core odo functions such as `odo create --devfile <devfile.yaml>`, `odo push`, `odo url create` should work with the devfile.
+    - PR tests on this repository will verify this functionality as well.
+
+2) Add a folder for the stack to `stacks/` in this repository.
```

Comment:
We may need to update this after we support the samples folder in the future. We can leave it as `stacks` folder only for the time being.

Reply:
Agreed. I'll update the issue that adds support for `samples` to mention this.